mysql-test/suite/parts/t/mdev-21027.test

```diff
@@ -0,0 +1,23 @@
+#
+# MDEV-21027 Assertion `part_share->auto_inc_initialized || !can_use_for_auto_inc_init()' failed in ha_partition::set_auto_increment_if_higher
+#
+
+--source include/have_partition.inc
+
+CREATE TABLE t1 (a INT AUTO_INCREMENT PRIMARY KEY) PARTITION BY HASH (a) PARTITIONS 3;
```

Comment:
MariaDB's default storage engine is MyISAM. So, the engine of the table `t1` above is MyISAM not InnoDB.

Reply:
Thanks, I misunderstood.